### ISSUE TITLE
No user defined rights on qgep_network

### ISSUE THREAD

**Issue by ponceta** [QGEP/QGEP#738]
**Describe the bug**
Only if you define qgep user as database owner and use it to connect will you to your qgep project. This is a bad practice in production since the rights are too elevated and the user will be able to alter the datamodel structure.

```
CREATE SCHEMA IF NOT EXISTS qgep_network
    AUTHORIZATION postgres;
```

**To Solve**
Provide finer permissions definition

**Expected behavior**
The user should be able to execute functions in qgep_network and truncate the node and segment tables

**Screenshots / data**
If applicable, add screenshots or data to help explain your problem.

**Desktop (please complete the following information):**
 - QGEP datamodel version 1.5.3

**Additional context**
Such rights management should enable the user to execute topology functions from the plugin without having admin rights.

**Comment by rbovard** [comment]
I confirm this issue and the need to find a solution to fix this.

Currently the user has an access to a button in QGEP plugin wich doesn't work.

**Comment by sjib** [comment]
How should this then be adapted?
https://github.com/QGEP/datamodel/blob/6aa60794729f24edcd9a7c6d7ad124a605dec4e9/12_1_roles.sql#L9
https://github.com/QGEP/datamodel/blob/6aa60794729f24edcd9a7c6d7ad124a605dec4e9/12_1_roles.sql#L13
etc

We already did a fix on this: https://github.com/QGEP/datamodel/pull/170/files

**Comment by rbovard** [comment]
Sorry, I confirm it's fixed in `1.5.5`

@ponceta you can close this issue.

**Comment by ponceta** [comment]
Demo data 1.5.6.1 -> Tried it with all delta applied, sql action is started correctly but there's something wrong when applying this on the demo data.

@olivierdalang Could you have a look at this fix on the demo-dataset?

The error when clicking on refresh symbology:

Error: Status 7 (ERREUR: une valeur NULL viole la contrainte NOT NULL de la colonne « id » DETAIL: La ligne en échec contient (null, wastewater_node, ch13p7mzWN003457, null, 0101000020080800001BCC6D5F77FA4441864B51FBE14A3341) CONTEXT: instruction SQL « INSERT INTO qgep_network.node(node_type, ne_id, geom) SELECT 'wastewater_node', n.obj_id, ST_Force2D(n.situation_geometry) FROM qgep_od.wastewater_node n » fonction PL/pgsql qgep_network.refresh_network_simple(), ligne 8 à instruction SQL )

To see also : https://github.com/QGEP/QGEP/issues/762 from @rbovard who sees that it is difficult to use topology functions after a topology refresh ? (cache issues?)
